shapedetector.py

```diff
@@ -2,54 +2,69 @@
 import imutils
 import numpy as np
 
-class ShapeDetector:
-    def __init__(self):
-        pass
+# Use otsu threshholding to find contours of shapes in image and count number of sides in contours
+# to determine which shapes and how many are present in the image
 
+class ShapeDetector:
+    # Take in image and determine the shapes with the number of sides
     def detect(self, c):
         shape = 'unidentified'
-        peri = cv2.arcLength(c, True)   # perimeter
-        approx = cv2.approxPolyDP(c, 0.04*peri, True)   # use RDP algorithm to simplify shape
+        # perimeter
+        peri = cv2.arcLength(c, True)   
+        # use RDP algorithm to simplify shape
+        approx = cv2.approxPolyDP(c, 0.04 * peri, True)   
 
+        # Check what shape is in image
+        # Shapes can only be square, triangle, line, or circle
         print(len(approx))
+        
+        AREARATIO = 0.4
```

Comment:
If your variables have multiple words you should put underscores between the words:
`AREARATIO` should be `AREA_RATIO`